.github/workflows/release.yml

```diff
@@ -0,0 +1,44 @@
+name: Publish
+
+on:
+  # publish from the Releases page:
+  release:
+    types: [published]
+  # publish from the Actions page:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version (e.g. 2.0.3)'
+        required: true
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v1
+    - name: Build tools
+      run: |
+          python -m pip install --upgrade pip
+          pip install build
+    - name: Build
+      run: |
+          python -m build --wheel --sdist
+    ### TODO: can the uploads be parallelized?
```

Comment:
I [don't think parallelization is currently possible](https://github.community/t/steps-in-parallel/16343) with `steps:`. :cry:

Reply:
Yeah... it is with jobs though?

I managed to make it work, but only [with](https://github.com/kousu/ANTs/blob/329ef595b00538abce365453eaec99371c2ec394/.github/workflows/ccpp.yml#L163)

>      uses: actions/upload-artifact@v2-preview

and

>     - uses: actions/download-artifact@v1

and a lot of copy-pasting.